dietpi/dietpi-software

```diff
@@ -8540,7 +8545,8 @@ _EOF_
             fastcgi_param front_controller_active true;
             fastcgi_pass php-handler;
             fastcgi_intercept_errors on;
-            fastcgi_request_buffering off;
+            # Disable because Jessie Nginx does not support that parameter
+	    #fastcgi_request_buffering off;
```

Comment:
Found the related report on nextcloud forum:
https://help.nextcloud.com/t/nc-not-running-on-nginx/656/14
We should keep this in mind, once we fully switch to stretch, or we can now `sed` this directive based on `$DISTRO` in script?